test-fixture/parse-2/expression/simple-expression/function-call/arguments/expression-1-arguments/arg-3.txt

```diff
@@ -0,0 +1,18 @@
+SELECT SQL_TSI_DAY(x, y, z)
+-----
+`!
+          (expression: ExtractFromDateTimeFunctionCall[7, 27]
+            (functionName: SQL_TSI_DAY[7, 18])
+            ({Expected exactly 1 item, found 3} arguments: Expression1_Arguments[18, 27]
```

Comment:
This is an example of a field check failing.

`SQL_TSI_DAY` expects only one argument.

However, we have three arguments here.